GithubJobApp/JobTableViewCell.swift

```diff
@@ -12,15 +12,16 @@ class JobTableViewCell: UITableViewCell {
     
     // MARK: Properties
     @IBOutlet weak var company: UILabel!    
-    @IBOutlet weak var companyLogo: UIView!
+    @IBOutlet weak var companyLogo: UINetImageView!
     @IBOutlet weak var title: UILabel!
     
-    
-    override func awakeFromNib() {
-        super.awakeFromNib()
-        
+    var job: Job? {
+        didSet {
+            guard let job = job else { return }
+            company.text = job.company
+            title.text = job.title
+            if let companyLogoUrl = job.companyLogoUrl { companyLogo.setImgFromUrl(companyLogoUrl) }
```

Comment:
Unless the code block between the {}s is completely empty, it always goes on it's own indented line, i.e.

```swift
if let companyLogoUrl = job.companyLogoUrl {
     companyLogo.setImgFromUrl(companyLogoUrl)
}
```